Liroy/bootstrap.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
The following shebang make more sense in a multi distribution environment:
`#!/usr/bin/env bash`